tensorflow_probability/python/experimental/mcmc/sample_discarding_kernel.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2020 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Sample Discarding Kernel for Thinning and Burn-in"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+
+# Dependency imports
+import tensorflow.compat.v2 as tf
+from tensorflow_probability.python.experimental.mcmc import step_kernel
```

Comment:
Does doing `from tensorflow_probability.python.experimental import mcmc; ... mcmc.step_kernel` work? If so, that's the style preferred by the style guide.